src/NServiceBus.Persistence.ServiceFabric/StaticVersions.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace NServiceBus.Persistence.ServiceFabric
+{
+    using System;
+    using System.Diagnostics;
+    using System.Reflection;
+
+    static class StaticVersions
```

Comment:
This class is also used to calculate the current version, see usage in the `RuntimeSagaInfo`